shared/src/commonTest/kotlin/com/mbta/tid/mbta_app/model/PatternSortingTest.kt

```diff
@@ -1,430 +1,12 @@
 package com.mbta.tid.mbta_app.model
 
-import com.mbta.tid.mbta_app.model.NearbyStaticData.StaticPatterns
-import io.github.dellisd.spatialk.geojson.Position
 import kotlin.test.BeforeTest
-import kotlin.test.Test
-import kotlin.test.assertEquals
-import kotlin.time.Duration.Companion.minutes
-import kotlinx.datetime.Clock
 
 class PatternSortingTest {
```

Comment:
There were no group-by-direction tests here, so this class is now empty; if it would make sense to port some of the tests instead of deleting them, I can take a look at doing that.

Reply:
That was an oversight on my part when adding the route card pattern sorting functionality! I think it would be nice to have at least the basics covered in a test here, even if just one test for each of the public functions.